contentcuration/contentcuration/models.py

```diff
@@ -361,6 +369,18 @@ def get_default(self):
         return result
 
 
+class MPTTTreeIDManager(models.Model):
+    """
+    Because MPTT uses plain integers for tree IDs and does not use an auto-incrementing field for them,
+    the same ID can sometimes be assigned to two trees if two channel create ops happen concurrently.
+
+    We resolve this by creating a dummy table and using its ID as the tree index to take advantage of the db's
+    concurrency-friendly way of generating sequential integer IDs. There is a custom migration that ensures
+    that the number of records (and thus id) matches the max tree ID number when this table gets added.
+    """
+    current = models.BooleanField(default=False)
```

Comment:
Don't see this field referenced anywhere except in the migration, so could perhaps drop it to make the table even leaner?

Reply:
Ah, yeah, good point, this was being used at one point but then realized I didn't need it, but forgot to remove it! Thanks!